recipes/VPRTempo/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "vprtempo" %}
+{% set version = "1.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/VPRTempo-{{ version }}.tar.gz
+  sha256: 2660af11c463893b22353967e7fc14b502b3b6d86331d66f47e33a43f11519aa
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
```

This should be in sync.
```suggestion
    - python >=3.6
```

---

Thanks fixed it all up, should be good now